src/Variable.ts

```diff
@@ -0,0 +1,65 @@
+import { fabric } from "fabric";
+import { makeLine } from "./Utils";
+import { Pointer } from "./Pointer";
+import Config = require("./Config");
+import { Node } from "./Node";
+
+export class Variable {
+    public name: string;
+    public value: Pointer;
+    private representation: fabric.Group;
+
+    constructor(name: string, canvas: fabric.Canvas) {
+        this.name = name;
+        this.value = new Pointer(this, canvas);
+
+        const text = new fabric.IText(this.name, {
+            fill: '#black',
+            evented: false,
+            selectable: false
+        });
+
+        this.representation = new fabric.Group([text], {
+            hasControls: false,
+            hasBorders: false,
+            hoverCursor: "grab",
+            moveCursor: "grabbing",
+        });
+
+        canvas.add(this.representation);
+        this.representation.center();
+    }
+
+    public draw(): void {
+        // Draw the pointer
```

Comment:
I don't think this comment is necessary, its very clear what is going on in the code here.